content/en/real_user_monitoring/explorer/group.md

```diff
@@ -11,7 +11,7 @@ further_reading:
 
```

Comment:
For this page, could we also update the screenshots where "Application ID" is in the query? That facet shouldn't be used as much now that we have the app selector

Reply:
I had missed these. Thanks. Fixed!